src/runtime/runtime.ts

```diff
@@ -56,37 +60,37 @@ export type RuntimeArcOptions = Readonly<{
   modality?: Modality;
 }>;
 
+
+// TODO(sjmiles): weird layering here due to dancing around global state (working on it)
+let staticMemoryProvider;
+const initDrivers = () => {
+  VolatileStorageKey.register();
+  staticMemoryProvider = new SimpleVolatileMemoryProvider();
+  RamDiskStorageDriverProvider.register(staticMemoryProvider);
+};
+
+initDrivers();
+
+const nob = Object.create(null);
+
 @SystemTrace
 export class Runtime {
   public context: Manifest;
   public readonly pecFactory: PecFactory;
+  public readonly loader: Loader | null;
   private cacheService: RuntimeCacheService;
-  private loader: Loader | null;
   private composerClass: typeof SlotComposer | null;
   private memoryProvider: VolatileMemoryProvider;
   readonly storageManager: StorageEndpointManager;
   readonly arcById = new Map<string, Arc>();
 
-  /**
-   * Call `init` to establish a default Runtime environment (capturing the return value is optional).
-   * Systems can use `Runtime.getRuntime()` to access this environment instead of plumbing `runtime`
-   * arguments through numerous functions.
-   * Some static methods on this class automatically use the default environment.
-   */
-  static init(root?: string, urls?: {}): Runtime {
-    const map = {...Runtime.mapFromRootPath(root), ...urls};
-    const loader = new Loader(map);
-    const pecFactory = pecIndustry(loader);
-    const memoryProvider = new SimpleVolatileMemoryProvider();
-    const runtime = new Runtime({
-      loader,
-      composerClass: SlotComposer,
-      pecFactory,
-      memoryProvider
-    });
-    RamDiskStorageDriverProvider.register(memoryProvider);
-    VolatileStorageKey.register();
-    return runtime;
+  static resetDrivers(noDefault?: true) {
```

Comment:
`noDefault` isn't a particularly obvious arg name, particularly given its effect (what does "default" have to do with initialising the drivers?)

Reply:
`resetDrivers` is going away ASAP. This was just a stopgap thing.